net.bartkessels.getit.json

```diff
@@ -0,0 +1,75 @@
+{
+	"app-id": "net.bartkessels.getit",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
```

Comment:
Does it specifically require something from 3.24 or is that just what it was tested on?

Reply:
No, it doesn't require anything specific from 3.24 it's only tested with that version